pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = [
     "ninja",
     "packaging",
     "setuptools",
-    "torch >= 2.0.0",
+    "torch == 2.0.1",
```

Comment:
Will `torch >= 2.0.0, < 2.1.0` be better? It seems that `torch == 2.0.0` works as well.

Reply:
Thanks for the suggestion. I think this is good since `xformers` requires torch 2.0.1

> (RECOMMENDED, linux & win) Install latest stable with pip: Requires [PyTorch 2.0.1](https://pytorch.org/get-started/locally/)